index.bs

```diff
@@ -569,13 +582,21 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |issuedRequests| and |currentlyAvailableAuthenticators| be new [=ordered sets=].
 
-1. For each |authenticator| currently available on this platform, if
-    <code>|options|.{{MakeCredentialOptions/attachment}}</code> is [=present|not present=] or its value matches
-    |authenticator|'s [=attachment modality=], [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|.
-
-1. If |currentlyAvailableAuthenticators| [=list/is empty=], return a {{DOMException}} whose name is
+1. If |currentlyAvailableAuthenticators| [=list/is empty=], return a {{DOMException}} whose name is 
```

Comment:
|currentlyAvailableAuthenticators| won't be empty at this point. At this point, the client goes down to search and find all the authenticators at the platform. Then the client checks to see if any of them meets the criteria set by the RPs. 
This way, if an RP sees NotFoundError, the RP knows there's no authenticator whatsoever, If the RP sees ConstraintError, the RP knows it hasn't set the right constraint yet or its constraint is limiting. If the RP sees QuotaExceededError, the RP knows the authenticator doesn't have storage in it.